js/ad_manager_videoplaza.js

```diff
@@ -527,13 +463,17 @@ OO.Ads.manager(function(_, $) {
      * @param {object} ad The ad object to cancel.
      */
     this.cancelAd = function(ad) {
-      //TODO: cancel linear ads
-      var overlayContainer = this.amc.ui.rootElement.find(this.OVERLAY_AD_CONTAINER);
-      if (overlayContainer) {
-        overlayContainer.remove();
-        this.lastOverlayAd = null;
-        if (_.isFunction(this.adEndedAMCCallback)) {
-          this.adEndedAMCCallback();
+      if (!this.amc || !this.amc.ui) {
+        return;
```

Comment:
Because amc ui contains a lot of variables that are needed like startTime, so if it is null that means AMC is broken or initialized properly.